contributing.md

```diff
@@ -245,7 +245,7 @@ npm run test:e2e -- --wordpress-base-url=https://my-amp-dev-site.local
 For debugging purposes, you can also run the E2E tests in non-headless mode:
 
 ```bash
-HEADLESS=false npm run test:e2e
+npm run test:e2e:interactive
```

Comment:
Maybe we can have `PUPPETEER_SLOWMO=200 PUPPETEER_HEADLESS=false npm run test:e2e` in the documentation as well in case someone would want to set their own SLOWMO time?

Reply:
Done in 5554749ac8d396cb555d04cb717a89769aeef102